js/example/example-app.js

```diff
@@ -1,5 +1,5 @@
 const {VCRenderer} = require("../lib/commonJs/vcrenderer.js")
-const {VC} = require('./sample-vc.js')
-VCRenderer.renderSVG(VC).then(
-    (response) => console.log("SVG Image->",response)
+const {InsuranceVC, MosipVCWithQR, MosipVCWithoutQR} = require('./sample-vc.js')
+VCRenderer.renderSVG(MosipVCWithoutQR).then(
+    (response) => console.log("::::::::::::::SVG Image->",response)
```

Comment:
should we have log as SVG template as we can't log image

Reply:
It is the final SVG Image gendered by replacing the SVG Template placeholders with VC data. Shall we have it as Final SVG XML ?

Reply:
@swatigoel It is the final SVG Image generated by replacing the SVG Template placeholders with VC data. Shall we have it as Final SVG XML ?